src/static/js/modules/loan-comparison/components/loan-output-table-row.js

```diff
@@ -0,0 +1,70 @@
+var React = require('react');
+var LoanOutputCell = require('./loan-output-table-cell');
+
+var LoanOutputRow = React.createClass({
+    render: function () {
+        var headingType = function (prop, type, label) {
+            if (type === 'primary') {
+                return (
+                    <LoanOutputRowPrimaryHeading prop={prop} label={label} />
+                );
+            } else {
+                return (
+                    <th scope="row">
+                        <h5>
+                            {label}&nbsp;
+                            <span className="lc-tooltip" data-toggle="tooltip" role="tooltip" data-original-title="" title=""><span className="cf-icon cf-icon-help-round"></span></span>
+                        </h5>
+                    </th>
+                );
+            }
+        };
+        var loans = this.props.loans.map(function (loan) {
+            return (
+                <LoanOutputCell loan={loan} prop={this.props.prop} resultType={this.props.resultType} />
+            )
```

Comment:
If we're not using it to generate classes, we could probably create the cells here instead of in a LoanOutputCell component.